cli/src/main/java/org/crsh/cli/descriptor/OptionDescriptor.java

```diff
@@ -101,7 +101,7 @@ public OptionDescriptor(
       annotation);
 
     //
-    if (getMultiplicity() == Multiplicity.MULTI && getType() == ValueType.BOOLEAN) {
+    if (Multiplicity.MULTI.equals(getMultiplicity()) && ValueType.BOOLEAN.equals(getType())) {
```

Comment:
This is an enum, and I discussed the situation with ValueType above.  With enums, what are you gaining by using the equals() method?  The JVM will arrive at the same evaluation.